drivers/bluetooth/hci/h4.c

```diff
@@ -278,7 +278,6 @@ static size_t h4_discard(const struct device *uart, size_t len)
 static inline void read_payload(void)
 {
 	struct net_buf *buf;
-	uint8_t evt_flags;
 	int read;
```

Comment:
Persssonal preferenssse: pleassse sssquassshh 🐍 all the `Remove references to ..` commits into one.